app/controllers/provider_interface/condition_statuses_controller.rb

```diff
@@ -0,0 +1,85 @@
+module ProviderInterface
+  class ConditionStatusesController < ProviderInterfaceController
+    before_action :set_application_choice
+    before_action :redirect_back_if_application_terminated
+    before_action :requires_make_decisions_permission
+    before_action :redirect_unless_feature_flag_enabled
+
+    def edit
+      @form_object = ConfirmConditionsWizard.new(condition_statuses_store, offer: @application_choice.offer)
+      @form_object.save_state!
+    end
+
+    def confirm
+      @form_object = ConfirmConditionsWizard.new(condition_statuses_store, attributes_for_wizard)
+      @form_object.save_state!
+
+      unless @form_object.valid?
+        track_validation_error(@form_object)
+        render action: :edit
+      end
+    end
+
+    def update
+      @form_object = ConfirmConditionsWizard.new(condition_statuses_store, offer: @application_choice.offer)
+
+      if @form_object.valid?
+        handle_individual_status_transitions
+
+        redirect_to provider_interface_application_choice_path(@application_choice)
+      else
+        track_validation_error(@form_object)
+        redirect_to action: :edit
+      end
+    end
+
+  private
+
+    def redirect_back_if_application_terminated
+      if ApplicationStateChange::TERMINAL_STATES.include?(@application_choice.status.to_sym)
+        redirect_back(fallback_location: provider_interface_application_choice_path(@application_choice))
+      end
+    end
+
+    def redirect_unless_feature_flag_enabled
+      return if FeatureFlag.active?(:individual_offer_conditions)
+
+      redirect_to provider_interface_application_choice_path(@application_choice)
+    end
+
+    def attributes_for_wizard
+      condition_statuses_params.merge(offer: @application_choice.offer)
+    end
+
+    def condition_statuses_params
+      params
+        .require(:provider_interface_confirm_conditions_wizard)
+        .permit(statuses: {})
+    end
+
+    def condition_statuses_store
+      key = "condition_statuses_store_#{current_provider_user.id}_#{@application_choice.id}"
+      WizardStateStores::RedisStore.new(key: key)
+    end
+
+    def handle_individual_status_transitions
+      ProviderInterface::SaveConditionStatuses.new(
+        actor: current_provider_user,
+        application_choice: @application_choice,
+        statuses_form_object: @form_object,
+      ).save!
+
+      flash[:success] = success_message
```

Comment:
Can we move this to the update action instead, so it's easier to see where that's setup.